staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go

```diff
@@ -74,12 +82,26 @@ func (m *PathRecorderMux) trackCallers(path string) {
 // refreshMuxLocked creates a new mux and must be called while locked.  Otherwise the view of handlers may
 // not be consistent
 func (m *PathRecorderMux) refreshMuxLocked() {
-	mux := http.NewServeMux()
+	newMux := mux.NewRouter().StrictSlash(true)
```

Comment:
> This adds redirect behavior we didn't have before. Currently, requesting any of the discovery paths with or without slashes returns expected data with an HTTP 200. This makes all slashed forms send a 301 to the unslashed form.

That's neat.  Guess the `kubectl` client handled that redirect automatically for me.  Thing we care about?